python/ptest/cases/dcdc_checkout_case.py

```diff
@@ -0,0 +1,100 @@
+from .base import SingleSatOnlyCase
+
+class DCDCCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("DCDC cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state("pan.cycle_no")
+
+        adcs_cmd = simulation.flight_controller.read_state("dcdc.ADCSMotor_cmd")
+        sph_cmd = simulation.flight_controller.read_state("dcdc.SpikeDock_cmd")
```

Comment:
To make this test case prettier maybe you can define class members that make the state fields a property of the class:

```
@property
def adcs_motor_cmd(self): 
    return self.sim.flight_controller.read_state("dcdc.ADCSMotor_cmd")
    
    # setting the values     
    @value.setter 
    def adcs_motor_cmd(self, value): 
        assert(value == True or value == False)
        self.sim.flight_controller.write_state("dcdc.ADCSMotor_cmd", true)
```

Then reading and writing the state field is as simple as doing `x = self.adcs_motor_cmd` and `self.adcs_motor_cmd = "whatever"`. It reduces the clutter in the test file.

Reply:
@afoarce I recommend this too for future testcase development. Not sure how applicable it is to Gomspace currently but there's probably some room for prettification there.